pkg/adaptor/cloud/vsphere/provider.go

```diff
@@ -295,5 +295,7 @@ func (p *vsphereProvider) DeleteInstance(ctx context.Context, instanceID string)
 }
 
 func (p *vsphereProvider) Teardown() error {
+	logger.Printf("Logout user %s", p.serviceConfig.UserName)
+	DeleteGovmomiClient(p.gclient)
```

Comment:
Hi @caavery !

The `DeleteGovmomicClient()` function (https://github.com/caavery/cloud-api-adaptor/blob/939c2394bb17aef826a7207bdc1f552603c850c5/pkg/adaptor/cloud/vsphere/vsphere.go#L78) has any error logged but an error object is not return. Why?

I wondering if it wouldn't be important to raise any error through the `Teardown()`'s return.

Reply:
Yes I think you are right. Fixing it now.

Thanks!